pkg/cri/resource-manager/resource-manager.go

```diff
@@ -118,6 +119,10 @@ func NewResourceManager() (ResourceManager, error) {
 	}
 
 	if err := m.setupRelay(); err != nil {
+		pid, _ := pidfile.Read()
```

Comment:
Hmm, shouldn't we do `ProcessExists()` here instead`

Reply:
Hmm... not sure, but maybe. Probably. The idea here was to try reading out the process idea from the PID file and print it out if we fail to start up. But it is true that it does not make much sense to print the process ID of a process if it does not exist, so maybe it should indeed be verified.

Reply:
Updated to keep PID file open and to use (`OwnerPid()`, the updated equivalent of) `ProcessExists()` to read the PID upon failure to start up.